index.html

```diff
@@ -1,134 +1,368 @@
+    </div>
+
+    <!-- step 2 -->
+    <div class='mt-3 d-none' id='collect-data-step'>
+      <p data-i18n="[html]step2.desc" class="h3 text-info"></p>
+      <div>
+        <!-- categories -->
+        <div class="row">
+          <div class="form-group col">
+            <label for='category' data-i18n="step2.catlabel"></label>
+            <input id='category' data-placeholder='' class="w-100" data-i18n="[data-placeholder]step2.catplaceholder" />
+          </div>
+          <div class="form-group col-sm-7">
+            <label for='categoryalt' data-i18n="step2.catalt"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-convenience-store"></i></span>
+              </div>
+              <input id='categoryalt' type='text' placeholder='' class="form-control" data-i18n="[placeholder]step2.cataltplaceholder" />
+            </div>
+          </diV>
+        </div>
+
+        <hr>
+        <p data-i18n="step2.addressalt" class="mt-3"></p>
+        <!-- address 1 -->
+        <div class="form-row">
+          <div class="form-group col-md-5">
+            <label for='addressalt' data-i18n="step2.street"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-round-pushpin"></i></span>
+              </div>
+              <input id='addressalt' type='text' placeholder='' class="form-control" data-i18n="[placeholder]step2.addressaltplaceholder" />
+            </div>
+          </div>
+          <div class="form-group col-md-2">
+            <label for="housenumber" data-i18n="step2.housenumber"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-keycap-10"></i></span>
+              </div>
+              <input type="number" class="form-control" id="housenumber" data-i18n="[placeholder]step2.housenumberplaceholder">
+            </div>
+          </div>
+
+          <div class="form-group col-md-3">
+            <label for="city" data-i18n="step2.city"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-cityscape"></i></span>
+              </div>
+              <input type="text" class="form-control" id="city" data-i18n="[placeholder]step2.cityplaceholder">
+            </div>
+          </div>
+          <div class="form-group col-md-2">
+            <label for="postcode" data-i18n="step2.postcode"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-input-numbers"></i></span>
```

Comment:
twa-postbox?

Reply:
Si, anche secondo me è più adatta in effetti. Inserisco.